src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day-sat calendar--month-length-31">
```

Comment:
https://mate-academy.github.io/fe-program/css/typical-bem-mistakes

pay attention to modifiers